app/views/rating/applications/_application.html.slim

```diff
@@ -18,8 +18,8 @@ tr
   td.text-left = application.flags.map { |f| content_tag(:span, f, class: 'label label-default') }.join('<br>').html_safe
   td.text-center = icon('check') if application.application_data['signed_off_at_project1']
   td.text-center = icon('check') if application.application_data['signed_off_at_project2']
-  td.text-center = icon('check') if application.application_data['mentor_fav_project1']
-  td.text-center = icon('check') if application.application_data['mentor_fav_project2']
+  td.text-center = icon('heart') if application.application_data['mentor_fav_project1'] == 'true'
```

Comment:
Sound defensive and *"future-proof"* love this 💚 
I have it in several places, will update it.

Does it need to be within this PR? It does not directly impact rating, so I could also do it a day later...?